content/docs/sidebar.json

```diff
@@ -140,6 +140,16 @@
       }
     ]
   },
+  {
+    "slug": "how-to",
```

Comment:
let's not do this a top section?

Reply:
does not look like it deserves to be

Reply:
Sure. I put it under the User Guide now (and moved the **Update Tracked Files** guide into **How To**).

Reply:
(...and **Troubleshooting**.)